app/utils.py

```diff
@@ -1253,11 +1253,14 @@ def generate_token(self, user_id, action_id):
             action_time)
 
     def verify_token(self, token, user_id, action_id):
```

Comment:
Is there no django or python utility to do this for you?

Reply:
Sent you an email about this.